src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -138,6 +138,8 @@ public enum OperationKind
         NameOfExpression = 0x123,
         /// <summary>Indicates an <see cref="ITupleExpression"/>.</summary>
         TupleExpression = 0x124,
+        /// <summary>Indicates an <see cref="IDynamicMemberReferenceExpression"/>.</summary>
+        DynamicAccessExpression = 0x125,
```

Comment:
We should name this as `DynamicMemberReferenceExpression` for consistency with the interface and other XXXReferenceExpression Operation kinds.